ansible/roles_ocp_workloads/ocp4_workload_mad_roadshow/tasks/workload.yml

```diff
@@ -298,6 +298,23 @@
       rhv_oracle_vm_database_password: "{{ ocp4_workload_mad_roadshow_oracle_db_password }}"
       rhv_oracle_vm_database_admin_password: "{{ ocp4_workload_mad_roadshow_oracle_dba_password }}"
 
+- name: Deploy RedHat OpenShift Data Science
+  when: ocp4_workload_mad_roadshow_rhods_setup | bool
+  include_role:
+    name: install_operator
+  vars:
+    install_operator_action: install
+    install_operator_name: "{{ ocp4_workload_mad_roadshow_rhods_operator_name }}"
+    install_operator_namespace: redhat-ods-operator
+    install_operator_channel: "{{ ocp4_workload_mad_roadshow_rhods_channel }}"
+    install_operator_automatic_install_plan_approval: "{{ ocp4_workload_mad_roadshow_rhods_automatic_install_plan_approval | default(true) }}"
+    install_operator_starting_csv: "{{ ocp4_workload_mad_roadshow_rhods_install_operator_starting_csv }}"
+    install_operator_manage_namespaces: []
+    install_operator_catalogsource_setup: "{{ ocp4_workload_mad_roadshow_rhods_use_catalogsource_setup | default(false)}}"
+    install_operator_catalogsource_name: "{{ ocp4_workload_mad_roadshow_rhods_catalogsource_name | default('') }}"
+    install_operator_catalogsource_image: "{{ ocp4_workload_mad_roadshow_rhods_catalogsource_image | default('') }}"
+    install_operator_catalogsource_image_tag: "{{ ocp4_workload_mad_roadshow_rhods_catalogsource_image_tag | default('') }}"
```

Comment:
Those defaults are not necessary and actually confusing, since you have defaults in `defaults/main.yml` of the role.